site/package.json

```
@@ -11,15 +11,18 @@
     "docs:build": "npm run docs:vuepress && npm run docs:sitemap",
     "docs:vuepress": "vuepress-vite build docs",
     "docs:fmt": "deno fmt",
-    "docs:sitemap": "sscli -b https://grammy.dev -r docs/.vuepress/dist --format xml --no-robots --no-clean"
+    "docs:sitemap": "sscli -b https://grammy.dev -r docs/.vuepress/dist --format xml --no-robots --no-clean",
+    "docs:serve": "serve docs/.vuepress/dist",
+    "docs:deno": "deno upgrade"
```

What is this script supposed to do?

---

I see, that makes sense!

However, the `docs:` prefix is meant to scope things that are about the docs themselves. Invoking a tool that runs independently of the docs doesn't really belong under the same scope IMO.

```suggestion
    "deno:upgrade": "deno upgrade"
```

How about this?